examples/minimal/packages/contracts/mud.config.ts

```diff
@@ -1,6 +1,8 @@
-import { mudConfig, resolveTableId } from "@latticexyz/world/register";
+import { mudConfig } from "@latticexyz/world/register";
+import { resolveTableId } from "@latticexyz/config";
 
 export default mudConfig({
+  namespace: "",
```

Comment:
isn't this the default?

Reply:
yes it is, good catch! added this in some intermediate version where importing `mudConfig` didn't work yet